easybuild/easyconfigs/p/Pillow/Pillow-3.2.0-foss-2016a-Python-3.5.1.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'PythonPackage'
+
+name = 'Pillow'
+version = '3.2.0'
+versionsuffix = '-Python-%(pyver)s'
```

Comment:
@boegel for two reasons. This PR is based on `foss` toolcahin and not `intel`, and its needs `Python3` and not `Python2` ;)